### ISSUE TITLE
smpp-simulator get wrong slf4j version

### ISSUE THREAD

**Issue by nhanth87** [RestComm/smscgateway#68]
in the parent pom.xml file, version of slf4j-api was 1.7.10 but after compile, smpp-simulator got version 1.5.6 

```
ls tools/smpp-simulator/bootstrap/target/simulator-smpp/lib/
ch-commons-charset-3.0.2.jar  java-getopt-1.0.9.jar  map-api-3.0.1336.jar   sccp-api-3.0.1336.jar     smpp-server-ra-library-3.0.0-SNAPSHOT.jar
ch-commons-util-6.0.2.jar     javolution-5.5.1.jar   map-impl-3.0.1336.jar  slf4j-api-1.5.6.jar      smpp-simulator-3.0.0-SNAPSHOT.jar
ch-smpp-5.0.10-SNAPSHOT.jar   log4j-1.2.14.jar       netty-3.9.6.Final.jar  slf4j-log4j12-1.7.13.jar  smsc-common-library-3.0.0-SNAPSHOT.jar

```

and smpp-simulator failed to run due to some exception:
```
SLF4J: The requested version 1.6 by your slf4j binding is not compatible with [1.5.5, 1.5.6]
SLF4J: See http://www.slf4j.org/codes.html#version_mismatch for further details.
```

It's not clear why smpp-simulator taken slf4j-api 1.5.6.

Workaround:
Delete slf4j-api 1.5.6 in path/to/simulator-smpp/lib/ and replace with newer version (1.7.x)

**Comment by nhanth87** [comment]
Hi @eduardocalfaia did you try to reproduce the bug?
Do you need some help?

Br,
TN

**Comment by vetss** [comment]
We have such bad behaviour because we refer to slf4j lib with version 1.7.* but some other reffered lib refers to a version 1.5.6. I found the the simpliest way to fix it - refer to slf4j 1.5.6 too.

This is a fix for the problem:
https://github.com/vetss/smscgateway/commit/4bdab9123d5837023381b2a6f6af01f6e228a1b1

Another thing is that we may be need to remove slf4j lib from deploy/restcom-smsc-gateway/lib at all because jboss 5.1 has such lib on board already. This lib is also included in a diameter mux "TelScale-diameter-mux-6.2.1.327.sar" (we can remove it from that folder too).
